### PR TITLE
Fix crictl page

### PR DESCRIPTION
**Pull request by tengqm**
This PR fixes a link (at the bottom of the page) to reference.
It also fixes the indentation of the number lists, the incorrect syntax tag for code snippets.

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/crictl.md [hunk 1]
```diff
@@ -236,98 +246,103 @@ Using `crictl` to run a pod sandbox is useful for debugging container runtimes.
...
-      ```
+   ```json
+   {
+     "metadata": {
+       "name": "nginx-sandbox",
+       "namespace": "default",
+       "attempt": 1,
+       "uid": "hdishd83djaidwnduwk28bcsb"
+     },
+       "logDirectory": "/tmp",
```
Comment (kbhawkey) [inline content/en/docs/tasks/debug-application-cluster/crictl.md:259]: nit: `logDirectory` and `linux` fields should be aligned with `metadata`?

Comment (tengqm) [reply content/en/docs/tasks/debug-application-cluster/crictl.md:259]: yes, will fix.

**Comment by kbhawkey** [comment]
Thanks!
/lgtm

**Pull request merged**